js/ui/text_box/ui.text_editor.base.js

```diff
@@ -447,7 +448,7 @@ const TextEditorBase = Editor.inherit({
 
 
     _cleanLabelObservable: function() {
-        if(this._labelContainerElement) {
+        if(this._labelContainerElement && this._label.isVisible()) {
```

Comment:
This code does not unobserve label container if label became invisible in runtime.